backend/entityservice/serialization.py

```diff
@@ -158,12 +163,13 @@ def get_similarity_scores(filename):
     logger.info("Starting download stream of similarity scores.", filename=filename, filesize=details.size)
 
     try:
-        csv_data_stream = iterable_to_stream(mc.get_object(config.MINIO_BUCKET, filename).stream())
-
-        # Process the CSV into JSON
-        csv_text_stream = io.TextIOWrapper(csv_data_stream, encoding="utf-8")
+        sims_data_stream = mc.get_object(config.MINIO_BUCKET, filename)
+        # TODO: Below is an Anonlink 'private' API. It should be made
```

Comment:
Can you please create an issue in anonlink for this.

Reply:
Done: n1analytics/anonlink#191.